container/nginx/p11nethsm.conf

```diff
@@ -3,7 +3,7 @@ slots:
     description: Local HSM (docker)
     operator:
       username: "operator"
-      # password: "opPassphrase"
+      password: "operatoroperator"
     instances:
       - url: "https://192.168.3.161:8443/api/v1"
```

Comment:
Please update this line to `- url: "https://localhost:8443/api/v1"` to keep it consistent with the value of `HOST` in the generate script.